code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -158,6 +158,20 @@
 	buildstacktype = /obj/item/stack/sheet/wood
 	item_chair = /obj/item/chair/wood
 
+/obj/structure/chair/sofa/bamboo
+	name = "bamboo sofa"
+	resistance_flags = FLAMMABLE
+	max_integrity = 70
+	buildstackamount = 2
+	buildstacktype = /obj/item/stack/sheet/wood
+
+/obj/structure/chair/stool/bamboo
+	name = "bamboo stool"
+	resistance_flags = FLAMMABLE
+	max_integrity = 70
+	buildstackamount = 2
+	buildstacktype = /obj/item/stack/sheet/wood
+
```

Comment:
these items are already defined, you just need to add the variables to the types on line 414 of this file and 440 respectively.

Reply:
>:(
like this you mean?
```/obj/structure/chair/sofa/bamboo
	name = "Bamboo Bench"
	desc = "Not the most comfortable, but vegan!"
	icon_state = "bamboo_sofamiddle"
	color = null
	colorable = FALSE
	resistance_flags = FLAMMABLE
	max_integrity = 70
	buildstackamount = 2
	buildstacktype = /obj/item/stack/sheet/wood
```